js/data/data_source_aliases.d.ts

```diff
@@ -0,0 +1,8 @@
+import DataSource, { DataSourceOptions } from './data_source';
```

Comment:
I suggest moving this to the [js/data/data_source.d.ts](https://github.com/DevExpress/DevExtreme/blob/21_2/js/data/data_source.d.ts) module

Reply:
[js/data/index.d.ts](https://github.com/DevExpress/DevExtreme/blob/21_2/js/data/index.d.ts) is even a better place for it, I think